src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -71,11 +73,26 @@ public SingleClusterMonitor(Map<String, Object> props, String name) throws Excep
   @Override
   public void start() {
     _topicManagementService.start();
-    CompletableFuture<Void> completableFuture = _topicManagementService.topicManagementResult();
-    completableFuture.thenRun(() -> {
+    CompletableFuture<Void> topicPartitionResult = _topicManagementService.topicPartitionResult();
```

Comment:
I found the naming of the 2 CompletableFuture variables used here to be a little bit confusing. If I understand what the code is supposed to be doing correctly, I would rename `topicPartitionResult` to `topicPartitionResultReady` and `topicPartitionFuture` to `producerConsumerReady`. The `ready` part is just my personal style. But making two variable names have the prefix `topicPartition` is just a little unintuitive to me

Reply:
Please see the PR description bullet number (4): `4 - Simplify the usage of CompletableFuture<Void> by using only one of CompletableFuture<Void> instead of two CompletableFuture<Void>` We are getting rid of one of the Futures to simplify the understanding and make operationally more efficient. In the final product, we are only using Future for TopicPartition being ready in time.